app/components/works/edit/citation_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= tag.div data: { controller: 'citation' } do %>
+  <p><%= t('works.edit.fields.citation.help_text') %></p>
+
+  <%= tag.div class: 'form-check mb-4' do %>
+    <%= form.radio_button :auto_generate_citation, true, class: 'form-check-input', checked: true, data: { citation_target: 'autoGenerateCitation', action: 'click->citation#autoGenerateCitationSelected' } %>
+    <%= form.label :auto_generate_citation_true, t('works.edit.fields.citation.auto_citation'), class: 'form-check-label' %>
+  <% end %>
+
+  <%= tag.div class: 'form-check mb-4' do %>
+    <%= form.radio_button :auto_generate_citation, false, class: 'form-check-input', data: { citation_target: 'customCitation', action: 'click->citation#customCitationSelected' } %>
+    <%= form.label :auto_generate_citation_false, t('works.edit.fields.citation.custom_citation') %>
```

Comment:
Does this not get the `form-label` class before its associated input isn't checked?